modules/virt-creating-vm-custom-template.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-creating-vm-custom-template_{context}"]
+= Creating a virtual machine from a customized template
+
+Some templates require additional parameters, for example, a PVC with a boot source. You can customize select parameters of a template to create a virtual machine (VM).
+
+.Procedure
+
+. Click *Virtualization* > *Catalog* in the side menu.
+
++
+Optional: Filter the templates by operating system or workload profile.
+
+. Click the template and then click *Customize VirtualMachine* to modify the template, for example, by specifying a boot source, the disk size, or a DataSource to clone.
+
+. Click *Review and create VirtualMachine* and edit the template parameters.
+
+. Click *Create VirtualMachine*.
++
+The VM *Details* page is displayed with the provisioning status.
```

Comment:
`s/VM *Details*/*VirtualMachine details*`

And because the provisioning status doesn't last for long, maybe: 

`The *VirtualMachine details* page is displayed with the current status of the VM.`